notebooks/Zero_cost_Zero_time_Zero_shot_Financial_Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,2428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    print('The WOS implies that in average {:0.1f}% of the sentences in the financial sentiment analysis (FSA) dataset are rather simple.\n'.format(100-100*WOS))
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>